internal/federationdomain/clientregistry/clientregistry.go

```diff
@@ -27,6 +27,12 @@ import (
 // or a dynamic client defined by an OIDCClient CR.
 type Client struct {
 	fosite.DefaultOpenIDConnectClient
+
+	// Optionally provide a lifetime for ID tokens that result from authcode exchanges (initial logins)
+	// and refresh grants for this specific client. This will not impact the lifetime of ID tokens created
+	// via RFC8693 token exchange. When zero, the ID token lifetime will be determined by the defaults
+	// for the FederationDomain.
```

Comment:
Defaults for the `FederationDomain`? Or a global default?

Reply:
And should this be a pointer to indicate its optionality? That might be more clear than `Duration=0`.

Reply:
Updated to be a pointer, specifically `*int32`.

Reply:
It is currently a global default, but leaving it slightly vague in the docs gives us flexibility to change it in the future. If you prefer we could change the doc to say exactly how it currently works instead.